Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -18,8 +21,33 @@ public class ReportService : IReportService
         public ITaskSchedulerService TaskSchedulerService { get; set; }
 
         public JiraRequestContext JiraRequestContext { get; set; }
```

Comment:
This is a service class. It should be stateless. Remove JiraRequestContext from here. If you want to use it, get it from the instanceId, and after that pass it as a parameter